XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="MSBuild.Sdk.Extras/2.0.54">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;netstandard2.0;netstandard2.1;Xamarin.iOS10;MonoAndroid90;MonoAndroid10.0;Xamarin.TVOS10;Xamarin.WatchOS10;Xamarin.Mac20;tizen40</TargetFrameworks>
+    <TargetFrameworks>netstandard1.0;netstandard2.0;Xamarin.iOS10;MonoAndroid90;MonoAndroid10.0;Xamarin.TVOS10;Xamarin.WatchOS10;Xamarin.Mac20;tizen40</TargetFrameworks>
```

Comment:
I added .NET Standard 2.1 support because it doesn't require the following NuGet Packages. We can avoid adding these extra NuGet Packages for users on .NET Standard 2.1:

- System.Threading.Tasks.Extensions (Not needed in .NET Standard 2.1)
- System.Reflection.Emit.Lightweight (Not needed in .NET Standard 2.1)
- System.Reflection.Emit.Lightweight (Not needed in .NET Standard 2.1)